src/main.rs

```diff
@@ -26,7 +33,7 @@ struct Cli {
     #[clap(short, long, value_parser, default_value_t = false)]
     debug: bool,
 
-    #[clap(short, long, value_parser, env = "GALOY_JWT", hide_env_values = true)]
+    #[clap(short, long, value_parser, default_value = "None")]
     jwt: Option<String>,
```

Comment:
lets rename all `jwt`s to token.

Reply:
Addressed in 4c2d705d7fa44243e00efc4db61d265866c716a0